docs/product-business-resource-planner-guide/src/main/asciidoc/optimizer-hard-vs-soft-constraints-con.adoc

```diff
@@ -0,0 +1,26 @@
+[id='optimizer-hard-vs-soft-constraints-con']
+= A planning problem has constraints
```

Comment:
I presume, once optaplanner-docs is moved into kie-docs, these sections wouldn't need to be duplicated any more?

Reply:
that's right it will be a lot easier to just maintain the same changes in the same place, and we need to use this particular anchor type for topic reuse and to build properly.